defaults/main.yml

```diff
@@ -3,11 +3,18 @@
 
 mariadb_databases: []
 mariadb_users: []
-mariadb_root_password: ''
+mariadb_root_password: root
```

Comment:
Is it necessary to have a non-empty default value? The default installation leaves the root password blank, I would think it's best to stay consistent with that.

Reply:
The empty password dangerous. Except learning processes and some docker/openshift application we should set password. This is only my opinion of course, not the official suggestion.